### DISCUSSION TITLE
The time limit in the third phase.

### DISCUSSION THREAD

**Discussion by AlvinAi96** [PaddlePaddle/PaddleSpatial#285] [BaiduKDDCup2022] [answered]
Excuse me, Could you tell me how much the time limit in the current phase? 2hours?or 8hours? or 10hours? @ramber1836

**Comment by ramber1836** [comment, marked as answer]
Same as the 2nd phase. 
https://github.com/PaddlePaddle/PaddleSpatial/discussions/275
